src/Markdown.MAML/Renderer/RenderCleaner.cs

```diff
@@ -4,11 +4,10 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Text.RegularExpressions;
-using Microsoft.PowerShell.Commands;
 
 namespace Markdown.MAML.Renderer
 {
-    static class RenderCleaner
+    static public class RenderCleaner
```

Comment:
What is the reason for making it public? We should minimise exposure to classes that don't need to be called outside the assembly. As we add extensibility we want to minimise people taking a dependency on a class that is for internal use.

Reply:
Agree. I used line normalization from this class in tests. What would be a better way?